Jinja2 version 3 documentation build failure

```
(ads-deploy) PC98125:docs klauer$ ads-deploy docs ../*.sln
INFO:ads_deploy.docs:Rendering general template: index.rst -> index.rst
INFO:ads_deploy.docs:Rendering per-PLC template: {{tsproj.name}}_{{plc.name}}_summary.rst -> lcls-plc-tmo-optics_tmo_optics_summary.rst
INFO:ads_deploy.docs:Rendering per-PLC template: {{tsproj.name}}_{{plc.name}}_epics.rst -> lcls-plc-tmo-optics_tmo_optics_epics.rst
INFO:ads_deploy.docs:Rendering per-tsproj template: {{tsproj.name}}_nc.rst -> lcls-plc-tmo-optics_nc.rst
INFO:ads_deploy.docs:Rendering per-tsproj template: {{tsproj.name}}_boxes.rst -> lcls-plc-tmo-optics_boxes.rst
INFO:ads_deploy.docs:Rendering per-tsproj template: {{tsproj.name}}_pragmas.rst -> lcls-plc-tmo-optics_pragmas.rst
INFO:ads_deploy.docs:Rendering per-PLC template: {{tsproj.name}}_{{plc.name}}_source.rst -> lcls-plc-tmo-optics_tmo_optics_source.rst
INFO:ads_deploy.docs:Rendering per-tsproj template: {{tsproj.name}}_links.rst -> lcls-plc-tmo-optics_links.rst
INFO:ads_deploy.docs:Done


(ads-deploy) PC98125:docs klauer$ pip install --upgrade jinja2
Collecting jinja2
  Downloading Jinja2-3.0.1-py3-none-any.whl (133 kB)
     |████████████████████████████████| 133 kB 7.7 MB/s
Collecting MarkupSafe>=2.0
  Downloading MarkupSafe-2.0.1-cp37-cp37m-macosx_10_9_x86_64.whl (13 kB)
Installing collected packages: MarkupSafe, jinja2
  Attempting uninstall: MarkupSafe
    Found existing installation: MarkupSafe 1.1.1
    Uninstalling MarkupSafe-1.1.1:
      Successfully uninstalled MarkupSafe-1.1.1
  Attempting uninstall: jinja2
    Found existing installation: Jinja2 2.11.2
    Uninstalling Jinja2-2.11.2:
      Successfully uninstalled Jinja2-2.11.2
Successfully installed MarkupSafe-2.0.1 jinja2-3.0.1

(ads-deploy) PC98125:docs klauer$ ads-deploy docs ../*.sln
INFO:ads_deploy.docs:Rendering general template: index.rst -> index.rst
INFO:ads_deploy.docs:Rendering per-PLC template: {{tsproj.name}}_{{plc.name}}_summary.rst -> lcls-plc-tmo-optics_tmo_optics_summary.rst
INFO:ads_deploy.docs:Rendering per-PLC template: {{tsproj.name}}_{{plc.name}}_epics.rst -> lcls-plc-tmo-optics_tmo_optics_epics.rst
Traceback (most recent call last):
  File "/Users/klauer/mc/envs/ads-deploy/bin/ads-deploy", line 11, in <module>
    load_entry_point('ads-deploy', 'console_scripts', 'ads-deploy')()
  File "/Users/klauer/docs/Repos/ads-deploy/ads_deploy/__main__.py", line 99, in main
    func(**kwargs)
  File "/Users/klauer/docs/Repos/ads-deploy/ads_deploy/docs.py", line 404, in main
    write_file(template_path, render_args, 'per-PLC')
  File "/Users/klauer/docs/Repos/ads-deploy/ads_deploy/docs.py", line 386, in write_file
    template = jinja_env.get_template(template_path.name)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/loaders.py", line 553, in load
    return loader.load(environment, name, globals)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/loaders.py", line 137, in load
    code = environment.compile(source, name, filename)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/environment.py", line 750, in compile
    source = self._generate(source, name, filename, defer_init=defer_init)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/environment.py", line 684, in _generate
    optimized=self.optimized,
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 117, in generate
    generator.visit(node)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/visitor.py", line 40, in visit
    return f(node, *args, **kwargs)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 888, in visit_Template
    self.blockvisit(node.body, frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 449, in blockvisit
    self.visit(node, frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/visitor.py", line 40, in visit
    return f(node, *args, **kwargs)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 1262, in visit_For
    self.blockvisit(node.body, loop_frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 449, in blockvisit
    self.visit(node, frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/visitor.py", line 40, in visit
    return f(node, *args, **kwargs)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 1299, in visit_If
    self.blockvisit(node.body, if_frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 449, in blockvisit
    self.visit(node, frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/visitor.py", line 40, in visit
    return f(node, *args, **kwargs)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 1262, in visit_For
    self.blockvisit(node.body, loop_frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 449, in blockvisit
    self.visit(node, frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/visitor.py", line 40, in visit
    return f(node, *args, **kwargs)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 1582, in visit_AssignBlock
    self.pop_assign_tracking(frame)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/compiler.py", line 813, in pop_assign_tracking
    ref = frame.symbols.ref(name)
  File "/Users/klauer/mc/envs/ads-deploy/lib/python3.7/site-packages/jinja2/idtracking.py", line 81, in ref
    "Tried to resolve a name to a reference that was"
AssertionError: Tried to resolve a name to a reference that was unknown to the frame ('key')
```

--- a/conda-recipe/meta.yaml
+++ b/conda-recipe/meta.yaml
@@ -20,7 +20,7 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - jinja2 <3.0
+    - jinja2
     - pytmc >=2.6.9
     - typhos >=1.0
 

--- a/conda-recipe/meta.yaml
+++ b/conda-recipe/meta.yaml
@@ -21,7 +21,7 @@ requirements:
   run:
     - python >=3.6
     - jinja2
-    - pytmc >=2.6.9
+    - pytmc >=2.11.0
     - typhos >=1.0
 
 test:

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-pytmc>=2.6.9
+pytmc>=2.11.0
 typhos>=1.0
 jinja2
 versioneer

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
 pytmc>=2.6.9
 typhos>=1.0
-jinja2<3.0
+jinja2
 versioneer

